src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Main {
```

Comment:
The `Main` class should not have any empty lines at the start of its definition. Please remove the empty lines to comply with the coding standards.